app/assets/javascripts/application.js

```diff
@@ -14,6 +14,7 @@
 //= require abortcontroller-polyfill/dist/abortcontroller-polyfill-only.js
 //= require url-polyfill/url-polyfill.js
 //= require whatwg-fetch/dist/fetch.umd.js
+//= require helpers/fetch-content.js
```

Comment:
I notice this is the first 'helper', but we also have a 'modal' directory of things that are also 'helpful' in nature. How do you feel about rename just renaming `modal/ -> helpers/`?

Reply:
Merged them in 0f12071